src/Tools/Microsoft.VisualStudio.Templates.Editorconfig.Wizard/Utilities/VSHelpers.cs

```diff
@@ -78,6 +78,13 @@ public static (bool isSolutionLevel, string? path, string? language, object? sel
         foreach (UIHierarchyItem selItem in items)
         {
             selectedItem = selItem.Object;
+
+            // Check if selected item is the solution 
+            if (selectedItem is EnvDTE.Solution)
+            {
+                return (true, Path.GetDirectoryName(DTE.Solution.FullName), HasVisualBasicProjects() ? LanguageNames.VisualBasic : LanguageNames.CSharp, selectedItem);
```

Comment:
What happens if there's both C# and VB projects?

Reply:
In 2019 where this feature still  works, it just generates for C#. I am not going to change any functionality without customer input. 

![image](https://github.com/dotnet/templates/assets/9122518/9dd57afe-d8c9-4bd5-af8e-2c78ed0f230a)